views/error404.html

```diff
@@ -0,0 +1,15 @@
+% include('base.html', title="PyJudge")
```

Comment:
base is the parent template. Not the sub template. Perhaps `rebase` is what you wanted to use? See [this section of the docs](https://bottlepy.org/docs/dev/stpl.html#stpl.rebase)

Reply:
i am unable to understand how toh implement rebase for includin css in notfound

Reply:
that's ok.

take your time